lib/whitehall/asset_manager_and_quarantined_file_storage.rb

```diff
@@ -5,10 +5,14 @@ def store!(file)
   end
 
   def retrieve!(identifier)
-    asset_manager_file = Whitehall::AssetManagerStorage.new(uploader).retrieve!(identifier)
     quarantined_file = Whitehall::QuarantinedFileStorage.new(uploader).retrieve!(identifier)
 
-    File.new(asset_manager_file, quarantined_file)
+    if Whitehall.use_asset_manager
```

Comment:
I think this looks fine - another option might be to conditionally switch the storage engine used based on the feature flag in the Uploaders. But given that this can go away when the feature flag is enable it may not be worth the effort.

Reply:
We've now decided to remove the feature flag in this branch so this no longer applies.